src/Chapter2.hs

```diff
@@ -500,7 +538,13 @@ True
 >>> isThird42 [42, 42, 0, 42]
 False
 -}
-isThird42 = error "isThird42: Not implemented!"
+
+isThird42 :: [Int] -> Bool
+isThird42 []         = False
+isThird42 [_]        = False
+isThird42 [_,_]      = False
+isThird42 (_:_:42:_) = True
+isThird42 (_:_:_:_)  = False
```

Comment:
This is correct, however, could be simplified. You actually need to consider only two cases: when `42` is on third position and anything else:
```suggestion
isThird42 (_:_:42:_) = True
isThird42 _  = False
```